src/sentry/conf/server.py

```diff
@@ -434,11 +434,12 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     'sentry.tasks.process_buffer', 'sentry.tasks.reports', 'sentry.tasks.reprocessing',
     'sentry.tasks.scheduler', 'sentry.tasks.signals', 'sentry.tasks.store', 'sentry.tasks.unmerge',
     'sentry.tasks.symcache_update', 'sentry.tasks.servicehooks',
-    'sentry.tagstore.tasks',
+    'sentry.tagstore.tasks', 'sentry.tasks.assemble'
 )
 CELERY_QUEUES = [
     Queue('alerts', routing_key='alerts'),
     Queue('auth', routing_key='auth'),
+    Queue('assemble', routing_key='assemble'),
```

Comment:
cc @JTCunning I'm not sure if we need to do anything special anymore from ops to handle a new queue.

Reply:
No, we're good. If the task takes up a significant amount of resources, we'll isolate it with another pool of workers.